presto-main/src/main/java/com/facebook/presto/testing/LocalQueryRunner.java

```diff
@@ -722,7 +721,8 @@ public Lock getExclusiveLock()
                 subplan.getFragment().getPartitioningScheme().getOutputLayout(),
                 plan.getTypes(),
                 subplan.getFragment().getPartitionedSources(),
-                outputFactory);
+                outputFactory,
+                null);
```

Comment:
Prefer `Optional` instead of `null`.

Reply:
@nezihyigitbasi : In the original code, `null` was used as `ExchangeClientSupplier` .
I agree using `null` is not a good idea. I will just use a dumb `ExchangeClientSupplier` that always throws :)